client/my-sites/themes/theme-options.js

```diff
@@ -66,10 +91,10 @@ const activate = {
 	action: activateAction,
 	hideForTheme: ( state, themeId, siteId ) => (
 		! getCurrentUser( state ) ||
-		( isJetpackSite( state, siteId ) && isJetpackSiteMultiSite( state, siteId ) ) ||
+		isJetpackSiteMultiSite( state, siteId ) ||
 		isThemeActive( state, themeId, siteId ) ||
 		( isThemePremium( state, themeId ) && ! isPremiumThemeAvailable( state, themeId, siteId ) ) ||
-		( isJetpackSite( state, siteId ) && ! isThemeAvailableOnJetpackSite( state, themeId, siteId ) )
+		! isThemeAvailableOnJetpackSite( state, themeId, siteId )
```

Comment:
I think we need to retain the `isJetpackSite` criterion here. According to the JSDoc for [isThemeAvailableOnJetpackSite](https://github.com/Automattic/wp-calypso/blob/a70587157b33783ba4d5e0e4a1e136680d459d51/client/state/themes/selectors.js#L609-L624),

> Whether a given theme is installed or can be installed on a Jetpack site.